.github/workflows/ci.yml

```diff
@@ -31,6 +31,22 @@ jobs:
           python-version: '3.9'
           numpy-version: '1.19.*'
 
+        - os-image: ubuntu-latest
+          os-name: linux
+          docker-image: quay.io/pypa/manylinux2014_aarch64
+          python-version: '3.7'
+          numpy-version: '1.19.0'
```

Comment:
It's ok to use a wildcard for the patch version. That won't change any ABI guarantees and it's better that way to pick up security fixes.

Reply:
Done